src/sqlparser/src/keywords.rs

```diff
@@ -95,6 +95,7 @@ define_keywords!(
     AUTHORIZATION,
     AUTO,
     AVG,
+    BACKFILL_RATE_LIMIT,
```

Comment:
Why this need to be added as keyword?

Reply:
Probably does not need to be. I will remove it.